external/storm-eventhubs/pom.xml

```diff
@@ -1,113 +1,120 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
- Licensed to the Apache Software Foundation (ASF) under one or more
- contributor license agreements.  See the NOTICE file distributed with
- this work for additional information regarding copyright ownership.
- The ASF licenses this file to You under the Apache License, Version 2.0
- (the "License"); you may not use this file except in compliance with
- the License.  You may obtain a copy of the License at
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+	license agreements. See the NOTICE file distributed with this work for additional 
```

Comment:
Nitpick: Your editor seems to have replaced spaces with tabs, which has made the diff much larger than it needs to be. Could you reformat these files back to something closer to the original formatting?